compiler/ballerina-lang/src/main/java/io/ballerina/projects/Document.java

```diff
@@ -53,4 +58,53 @@ public SyntaxTree syntaxTree() {
     public TextDocument textDocument() {
         return this.documentContext.textDocument();
     }
+
+    /** Returns an instance of the Document.Modifier.
+     *
+     * @return  module modifier
+     */
+    public Modifier modify() {
```

Comment:
Better to change the API name to a noun instead of the verb, since this seems to be working as a getter

Reply:
This is the name used in classes in the syntax tree. Therefore, we have followed the same.